Assignment0/header.hpp

```diff
@@ -0,0 +1,22 @@
+
+#ifndef MATRIXVECTORPRODUCT_H
+#define MATRIXVECTORPRODUCT_H
+
+#include <vector>
+#include <stdexcept>
```

Comment:
This header is not used here. If you need this header in the cpp file, import it there.